sdk/eventhub/azure-eventhubs-checkpointstoreblob-aio/setup.py

```diff
@@ -66,7 +66,7 @@
     packages=find_packages(exclude=exclude_packages),
     python_requires=">=3.5.3",
     install_requires=[
-        'azure-storage-blob<=12.0.0b4,>=12.0.0b2',
+        'azure-storage-blob<=12.0.0b5,>=12.0.0b2',
```

Comment:
Let's change to "azure-storage-blob<=12.1,>=12.0.0b4" because azure-storage-blob is going to be GA soon and I guess the version will be 12.0.0 or 12.0.x if there is re-release for GA.